azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -305,6 +305,25 @@ func resourceArmKubernetesCluster() *schema.Resource {
 				},
 			},
 
+			"windows_profile": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"admin_username": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"admin_password": {
+							Type:         schema.TypeString,
+							Optional:     true,
```

Comment:
since this is a password can we also make this:

```suggestion
							Optional:     true,
							Sensitive: true,
```